src/test/regress/Makefile

```diff
@@ -137,6 +137,11 @@ check-multi-vg: all
 	--pg_ctl-timeout=360 --connection-timeout=500000 --valgrind-path=valgrind --valgrind-log-file=$(VALGRIND_LOG_FILE) \
 	-- $(MULTI_REGRESS_OPTS) --schedule=$(citus_abs_srcdir)/multi_schedule $(EXTRA_TESTS)
 
+check-columnar-vg: all
+	$(pg_regress_multi_check) --load-extension=citus --valgrind \
+	--pg_ctl-timeout=360 --connection-timeout=500000 --valgrind-path=valgrind --valgrind-log-file=$(VALGRIND_LOG_FILE) \
```

Comment:
Should we use a different file name for `--valgrind-log-file` (considering that we can do `make check-multi-vg check-columnar-vg`) ?

Reply:
Just add a ``columnar.`` prefix or something.

Reply:
Alternatively, I think it just appends to the file. If a successful run doesn't have too much noise, it might be fine to mix them.

Reply:
I introduced some arbitrary memory leaks and executed `make check-base-vg check-columnar-vg`, seems like log files overwrite each other, so I think we should have a prefix

Reply:
1da42f7a39fffb3ea52a74d761a6ebcf5eab8141